package.json

```diff
@@ -20,28 +20,28 @@
     "bootstrap": "5.2.3",
     "bootstrap-icons": "1.10.2",
     "browser-fs-access": "0.31.1",
-    "caniuse-lite": "1.0.30001435",
-    "d3": "7.6.1",
+    "caniuse-lite": "1.0.30001439",
+    "d3": "7.7.0",
     "file-saver": "2.0.5",
     "highcharts": "10.3.2",
-    "jquery": "3.6.1",
+    "jquery": "3.6.2",
     "jquery-ui-dist": "1.13.2",
     "js-cookie": "3.0.1",
     "list.js": "2.3.1",
     "lodash": "4.17.21",
     "lodash-es": "4.17.21",
     "luxon": "3.1.1",
     "moment": "2.29.4",
-    "moment-timezone": "0.5.39",
+    "moment-timezone": "0.5.40",
     "ms": "2.1.3",
     "murmurhash-js": "1.0.0",
     "naive-ui": "2.34.2",
-    "pinia": "2.0.27",
+    "pinia": "2.0.28",
     "pinia-plugin-persist": "1.0.0",
     "select2": "4.1.0-rc.0",
     "select2-bootstrap-5-theme": "1.3.0",
     "send": "0.18.0",
-    "shepherd.js": "10.0.1",
```

Comment:
Don't upgrade to a major version as there are breaking changes in 11.0.0 that needs to be addressed.

Reply:
Oops! Good catch. Was by accident.